RU/hardware.md

```diff
@@ -0,0 +1,14 @@
+
+# Hardware setup
+
+
+**Kaldi_NL**: [Official github repository](https://github.com/opensource-spraakherkenning-nl/Kaldi_NL)
+
+
+**Whisper**: [Official github repository](https://github.com/openai/whisper)
+
+
+**Wav2vec2.0-large-xlsr-53-dutch**: [Official github repository](https://huggingface.co/jonatasgrosman/wav2vec2-large-xlsr-53-dutch)
+
+
+For using Kaldi_NL and Whisper no GPUs are needed (but highly recommended).
```

Comment:
If possible, could you add the specifications of the hardware used during evaluation? For example, the GPU used, video memory capacity, CPU used, RAM capacity.